### PR TITLE
add icds-cas-sandbox to all reports

### PR DESCRIPTION
**Pull request by calellowitz**
@emord 
I am not fully convinced this will work, but I am optimistic
Ran:
```
ag -L icds-cas-sandbox custom/icds_reports/ucr/reports | grep json | xargs sed -i '' -e '/"icds-cas"/a\
\ \ \ \ "icds-cas-sandbox",'
```

### CODE REVIEW

File: custom/icds_reports/ucr/reports/other/it_issues_activity.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
         "reach-test",
         "icds-test",
         "icds-cas",
+    "icds-cas-sandbox",
```
Comment (emord) [inline custom/icds_reports/ucr/reports/other/it_issues_activity.json]: nit: indentation

**Review by emord — APPROVED** [review]
Should this also be run on `ucr/data_sources` ?

also I think:

`ag -L icds-cas-sandbox custom/icds_reports/ucr/reports | grep json`

could have been shorted to:

`ag -L icds-cas-sandbox custom/icds_reports/ucr/reports --json`

**Comment by calellowitz** [comment]
Most of the missing data sources were dashboard ones but doing it for all the others now.

**Review by emord — APPROVED** [review]

**Comment by mkangia** [comment]
@calellowitz 
Was it missing on all reports and data sources or only some?

**Pull request merged**